### DISCUSSION TITLE
Is 2.0 still in pre-release state?

### DISCUSSION THREAD

**Discussion by jcollum-nutrien** [artilleryio/artillery#2126] [Artillery]
The `2.0.0-3x` version numbers make it unclear.

**Comment by bernardobridge** [comment]
Hello @jcollum-nutrien,

No, `version 2` is the current maintained release, so you can update by following `2.0.0-x`. Thank you for the feedback though, we'll discuss that internally.

**Reply by jcollum-nutrien** [reply]
It's an odd numbering scheme, generally any version that isn't following x.x.x (where x is an integer only) I consider to be in some sort of pre-release state e.g. `2.0.0rc1` or `2.0.0alpha`. `-3x` is odd enough to make me wonder what's going on, which means I have to come here and ask before I go installing something that may be pre-release.

**Comment by bernardobridge** [comment]
Hi @jcollum-nutrien and others 👋

Starting from [Artillery v2.0.1](https://github.com/artilleryio/artillery/releases/tag/artillery-2.0.1), we'll be following more conventional numbering for our versions.
